karma.conf.js

```diff
@@ -16,7 +16,7 @@ module.exports = function(config) {
 
 		// list of files / patterns to load in the browser
 		files: [
-			'http://polyfill.webservices.ft.com/v1/polyfill.js?ua=safari/4',
+			'http://polyfill.webservices.ft.com/v2/polyfill.js',
```

Comment:
Can you add the gated flag: `?flags=gated` string, to the end of the Polyfill URL. We've had a couple of issues with PhantomJS when it loads the Polyfill service which using the gated flag is helping to prevent.